lib/sproxyd.js

```diff
@@ -81,7 +95,10 @@ class SproxydClient {
         }
         this.immutable = options.immutable || false;
         this.setCurrentBootstrap(this.bootstrap[0]);
-        this.httpAgent = new httpAgent.Agent();
+        this.httpAgent = new httpAgent.Agent({
+            freeSocketTimeout: 60 * 1000,
+            timeout: 2 * 60 * 1000,
```

Comment:
that [=the computation for timeout] is the default already, no?

Reply:
Yes, but I prefer it to be explicit, as httpagent is based on agentkeepalive, and it may not be clear if we rely on default values.